iot-hub/Quickstarts/ReadD2cMessages/Program.cs

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 // This application uses the Azure Event Hubs Client for .NET
-// For samples see: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/README.md
+// For samples see: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/eventhub/Azure.Messaging.EventHubs/samples/README.md
```

Comment:
the azure-sdk-for-net and azure-sdk-for-java both have been updated to use main as the defulat branch.

Reply:
Note that these links were still functional thanks to the redirecting functionality of GH